libobs/obs.h

```diff
@@ -517,6 +517,14 @@ EXPORT const char *obs_get_module_data_path(obs_module_t *module);
  */
 EXPORT void obs_add_module_path(const char *bin, const char *data);
 
+/**
+ * Adds a module to the list of modules allowed to load in safe mode.
+ * If the list is empty, all modules are allowed.
+ *
+ * @param  name  Specifies the module's name (filename sans extension).
+ */
+EXPORT void obs_add_safe_module(const char *name);
```

Comment:
Doesn't this make the function available to plugins as well? Meaning that _anybody_ can add safe modules to this list?

Reply:
Yes, that is a limitation of libobs, we don't really have a means of defining "UI only" APIs. But since third-party plugins aren't ever loaded, they cannot add themselves if that's what you were thinking.